### PR TITLE
Translation update 0.C

### PR DESCRIPTION
**Pull request by kevingranade**
Several updates to language handling as per http://smf.cataclysmdda.com/index.php?topic=4403.msg225130#msg225130

### CODE REVIEW

File: src/newcharacter.cpp [hunk 1]
```diff
@@ -1764,12 +1764,12 @@ int set_description(WINDOW *w, player *u, character_type type, int &points)
...
         werase(w_scenario);
         mvwprintz(w_scenario, 0, 0, COL_HEADER, _("Scenario: "));
-        wprintz(w_scenario, c_ltgray, g->scen->gender_appropriate_name(u->male).c_str());
+        wprintz(w_scenario, c_ltgray, _(g->scen->gender_appropriate_name(u->male).c_str()));
         wrefresh(w_scenario);
 
         werase(w_profession);
         mvwprintz(w_profession, 0, 0, COL_HEADER, _("Profession: "));
-        wprintz (w_profession, c_ltgray, u->prof->gender_appropriate_name(u->male).c_str());
+        wprintz (w_profession, c_ltgray, _(u->prof->gender_appropriate_name(u->male).c_str()));
```
Comment (BevapDin) [inline src/newcharacter.cpp]: Are you sure this is required? It seems the names are already translated via `pgettext` in `profession::load_profession`.

File: src/veh_interact.cpp [hunk 2]
```diff
@@ -1826,7 +1826,7 @@ void veh_interact::display_details( const vpart_info *part )
     if ( part->fuel_type != "NULL" ) {
         fold_and_print(w_details, line+4, col_1, ( vertical_menu ? column_width : details_w ), c_white,
                        _("Charge: <color_ltgray>%s</color>"),
-                       part->fuel_type.c_str());
+                       _(part->fuel_type.c_str()));
```
Comment (BevapDin) [inline src/veh_interact.cpp]: `fuel_type` is actually an identifier (an ammo type). In `veh_interact::do_refill`, it's translated like this:
```C++
ammo_name( vehicle_part_types[...].fuel_type ).c_str()
```

Some parts of the code seem to think it's an item type id.

File: src/newcharacter.cpp (hunk 1 above)
Comment (kevingranade) [inline src/newcharacter.cpp]: You're right, scenario was untranslated, but profession was fine, will revert.

**Comment by kevingranade** [comment]
Updated based on BevapDin's comments, thanks!

**Comment by Rivet-the-Zombie** [comment]
We're getting so much closer to 0.C, it's wonderful!

I'm looking forward to the end of this freeze.

**Pull request merged**